deploy/acm-policies/policy-cluster-admin.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: cluster-admin
+    namespace: openshift-managed-rbac-policies
+spec:
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: cluster-admin
+            spec:
+                object-templates:
+                    - complianceType: musthave
```

Comment:
just for knowledge: is this creating the cluster-admin(itself and its dependent resources) if not exist? 
Or it's just for throwing a compliance alert that the policy didn't meet the expected criteria?

Reply:
Yeah the Policy will Create the clusterrolebinding if it didn't exist already. if the complianceType set to inform, it would tell us that the clusterrolebinding is missing without creating it.